Firmware/messages.cpp

```diff
@@ -228,3 +228,17 @@ const char MSG_ADVANCE_K[] PROGMEM_N1 = "Advance K:"; ////c=13
 const char MSG_POWERPANIC_DETECTED[] PROGMEM_N1 = "POWER PANIC DETECTED"; ////c=20
 const char MSG_LCD_STATUS_CHANGED[] PROGMEM_N1 = "LCD status changed";
 const char MSG_UNKNOWN_CODE[] PROGMEM_N1 = "Unknown %c code: %s\n";
+
+// Common G-gcodes
+const char G1_E_F2700[] PROGMEM_N1 = "G1 E%-0.3f F2700";
+const char G28W0[] PROGMEM_N1 = "G28 W0";
```

Comment:
Just wondering, what would happen if we ommit these "0" argument values? Can we save just a few more bytes from this? So instead of "G28 W0", we use "G28 W". Same for the M701 and M702 a few lines lower

Reply:
Every char less saves bytes :wink:

Reply:
It save additional 2 bytes